requirements.txt

```diff
@@ -6,4 +6,5 @@ psycopg2
 flask
 flask-cors
 jsonify
-response
\ No newline at end of file
+response
+markdown
```

Comment:
Should add a version number to this since we have an open story to do so for the rest of them